pkgs/games/katago/default.nix

```diff
@@ -34,15 +35,24 @@ let
 
 in env.mkDerivation rec {
   pname = "katago";
-  version = "1.3.5";
+  version = "1.4.2";
 
   src = fetchFromGitHub {
     owner = "lightvector";
     repo = "katago";
     rev = "v${version}";
-    sha256 = "1625s3fh0xc2ldgyl6sjdjmpliyys7rzzkcys6h9x6k828g8n0lq";
+    sha256 = "0qdc9hgbzd175b2xkjs62dy6gyybcn9lf1mifiyhjbzjpgv192h4";
   };
 
+  # To workaround CMake 3.17.0's new buggy behavior wrt CUDA Compiler testing
```

Comment:
In general, when adding a patch, link the upstream issue and mention if/when the patch can be removed.

Reply:
That's fair, I'll edit the PR to include the two issues.

Reply:
The PR now has all relevant tracking issues.